ocis-pkg/registry/registry.go

```diff
@@ -68,28 +63,6 @@ func GetRegistry(opts ...Option) mRegistry.Registry {
 		case "memory":
 			_reg = memr.NewRegistry()
 			cfg.DisableCache = true // no cache needed for in-memory registry
-		case "kubernetes":
-			fmt.Println("Attention: kubernetes registry is deprecated, use nats-js-kv instead")
-			_reg = kubernetesr.NewRegistry()
```

Comment:
I still think the kubernetes registry is the right one to use in kubernetes. That we moved to nats-js-kv was due to bugs and misconfiguration. We are currently creating dozens of nats connections, eg in the gateway, when we could reuse one connection: every go micro http/grpc client creates an internal registry cache. A different problem, but it tells me that we should not drop support for the kubernetes registry, yet.